ui/pages/confirmations/components/confirm-page-container/confirm-page-container-navigation/confirm-page-container-navigation.component.js

```diff
@@ -36,9 +36,12 @@ const ConfirmPageContainerNavigation = () => {
     ...enumUnapprovedEncryptMsgsKey,
   ];
 
-  const enumUnapprovedTxs = Object.keys(unconfirmedTransactions).filter(
-    (key) => enumDecryptAndEncryptMsgs.includes(key) === false,
-  );
+  const enumUnapprovedTxs = Object.keys(unconfirmedTransactions)
```

Comment:
Why are we sorting here? And if we need sorting can't we use the `unconfirmedTransactionsListSelector` selector (that should already return the unconfirmed transactions sorted)